daemon/logger/splunk/splunk.go

```diff
@@ -366,6 +367,10 @@ func (l *splunkLoggerRaw) Log(msg *logger.Message) error {
 	message := l.createSplunkMessage(msg)
 
 	message.Event = string(append(l.prefix, msg.Line...))
+	// empty or whitespace-only mesages are not accepted by HEC
+	if strings.TrimSpace(message.Event.(string)) == "" {
```

Comment:
Wondering if it would be a problem to just check the `msg.Line`, and don't take the prefix into account; if line is empty, basically we'd just be logging the prefixes.

Just thinking out loud here, so happy to hear what others think